src/js/components/Accordion/accordion.stories.js

```diff
@@ -57,6 +72,28 @@ const SimpleAccordion = props => {
   );
 };
 
+const CustomAccordion = props => {
+  const { animate, multiple, ...rest } = props;
```

Comment:
could we destructure directly in the declaration?

`const CustomAccordion = ({animate, multiple, ...rest}) => (`

Reply:
I went ahead and changed the destructure and also added the box prop of height that way users can see that box does that that prop instead of a instyle which is already shown they are able to see it both ways!